### DISCUSSION TITLE
Homebrew's PHP is flagged as malicious by a single antivirus on VirusTotal

### DISCUSSION THREAD

**Discussion by tecknet000** [Homebrew/discussions#5069] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.4
ORIGIN: https://github.com/Homebrew/brew
HEAD: 1c612baa096c69f2fcb221c74e6f5b9979efdcee
Last commit: 5 days ago
Core tap JSON: 20 Jan 20:18 UTC
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :0
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 3.1.4 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: quad-core 64-bit sandybridge
Clang: N/A
Git: 2.34.1 => /bin/git
Curl: 7.81.0 => /bin/curl
Kernel: Linux 6.5.0-14-generic x86_64 GNU/Linux
OS: Ubuntu 22.04.3 LTS (jammy)
Host glibc: 2.35
/usr/bin/gcc: 11.4.0
/usr/bin/ruby: N/A
glibc: N/A
gcc@11: N/A
gcc: 13.2.0
xorg: N/A
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

PHP binaries from Homebrew are being flagged as malicious on VirusTotal, but I assume this is probably a false positive? Since all other antiviruses (including reputable names) don't flag it. Do you think it's a false positive?

I'm writing this from Linux, on which I have Homebrew installed, but I noticed the same thing on my Mac with a macOS PHP binary. I think I tried PHP 8.3, 8.2, and 8.1, and if I remember right, I think all of them were flagged as malicious by the same antivirus, "Kingsoft".

I have attached a picture. And here is the latest Homebrew PHP binary on VirusTotal: https://www.virustotal.com/gui/file/827907b466ffc11df283198e257814706104272362e685046ebf48c145c5c3fa

![php](https://github.com/Homebrew/discussions/assets/89277635/2cbc507e-23f6-4c1a-b573-2cd033081ac8)

**Comment by SMillerDev** [comment]
I'm not sure what you want as a response other than "we disagree"

**Comment by p-linnane** [comment]
This is a false positive. You can view the git history and see that this was built from source, using upstreams tarball, which is verified with a sha256 hash.
